script/setup

```diff
@@ -20,7 +20,7 @@ yarn dev | indent
 print_done
 
 notice "Installing shards"
-shards install --ignore-crystal-version | indent
+shards check --ignore-crystal-version || shards install --ignore-crystal-version | indent
```

Comment:
I'm not sure about this one. What is the `check` being used for? One thing I've noticed with `shards check` is if you change the `shard.yml` like removing a shard no longer used, the `shards check` command still passes.

Reply:
shards install always fetch shards, I used check to prevent this. But I not checked it for remove shards, it says "Dependencies are satisfied" when remove shards and change shard version, so better remove this check